Code/Engine/GameEngine/DearImgui/Implementation/DearImgui.cpp

```diff
@@ -314,7 +314,7 @@ void ezImgui::BeginFrame(const ezViewHandle& hView)
 
   ImGuiIO& cfg = ImGui::GetIO();
 
-  if (auto* pCVar = (ezCVarFloat*)ezCVar::FindCVarByName("Debug.TextScale"))
+  if (auto* pCVar = (ezCVarFloat*)ezCVar::FindCVarByName("App.TextScale"))
```

Comment:
The cvar should be cached somewhere. FindCVarByName does a linear search over all cvars which is not a good idea to do every frame.